recipes/ipyparallel/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set version = "5.0.1" %}
+
+package:
+  name: ipyparallel
+  version: {{ version }}
+
+source:
+  fn: ipyparallel-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/i/ipyparallel/ipyparallel-{{ version }}.tar.gz
+  md5: e771add417b0d1b98d3ae92099becce5
+
+build:
+  number: 0
+  script: python setup.py install
```

Comment:
My personal habit for conda packages is to use `pip install --no-deps .`, because it ensures two things:

- packages that *do* import setuptools unconditionally don't lay an egg
- packages that *don't* import setuptools for install (specifically to avoid the egg), can still have their setuptools extras_require, etc. (IPython does this)

I have a hard time thinking of any cases where `setup.py install` is preferable to `pip install .`.